kythe/go/extractors/config/default/mvn/mvn.go

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2018 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package mvn defines a default kythe.proto.ExtractionConfiguration for java
+// using Maven.
+package mvn
+
+import (
+	"io"
+	"strings"
+)
+
+const defaultMvnConfig = `{
```

Comment:
I'd leave out the `mvn` here: `defaultConfig` is sufficient (given we're already in package `mvn`).

Reply:
done